plugins/conf_maps/lib/api/KalturaConfMaps.php

```diff
@@ -105,17 +110,41 @@ public function validateForInsert($propertiesToSkip = array())
 		parent::validateForInsert($propertiesToSkip);
 	}
 
-	public function validateContent()
+	public function validateAndHandleContent()
 	{
-		$contentArray = json_decode($this->content, true);
-		if(!$contentArray)
+		$content = json_decode($this->content, true);
+		if (is_null($content))
+		{
+			throw new KalturaAPIException(KalturaErrors::CANNOT_PARSE_CONTENT, "Cannot JSON decode content", $this->content);
+		}
+
+		$filter = new KalturaConfMapsFilter();
+		$filter->relatedHostEqual = $this->relatedHost;
+		$filter->nameEqual = $this->name;
+		kApiCache::disableCache();
```

Comment:
Why did you need to disable api cache at this point

Reply:
since you are retrieving the full map and you might not have the updated content in your cache.